redaxo/src/addons/users/pages/users.php

```diff
@@ -172,6 +172,7 @@
         'user' => $user,
         'password' => $userpsw,
     ], true));
+    rex_user_session::getInstance()->removeSessionExceptCurrent($userId);
```

Comment:
Hier werden jetzt die User immer ausgeloggt, wenn man sie bearbeitet, auch wenn sich das Passwort nicht ändert.

Spricht etwas dagegen, den Call innerhalb von backend_login::changedPassword aufzurufen?

Reply:
> Hier werden jetzt die User immer ausgeloggt, wenn man sie bearbeitet, auch wenn sich das Passwort nicht ändert.

guter Punkt

> Spricht etwas dagegen, den Call innerhalb von backend_login::changedPassword aufzurufen?

ich möchte hier sicherstellen, dass wenn ich als User A von User B das Passwort ändere, dass alle Sessions invalide werden. Das wird ja schon in dem eigentlichen Session Handling von redaxo gemacht, hier möchte ich aber die dadurch obsoleten DB Einträge direkt loswerden.

Reply:
OK stimmt, dann passt es da nicht so gut rein.